### PR TITLE
[REVIEW] Fixing drop() error in `dask-cudf`

### PR DESCRIPTION
**Pull request by rjzamora**
Resolving [dask-cudf bug](https://github.com/rapidsai/dask-cudf/issues/158) when dropping a column. `dask` assumes the underlying function will accept an `errors` argument.

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

File: python/cudf/dataframe/dataframe.py [hunk 1]
```diff
@@ -917,14 +917,17 @@ def add_column(self, name, data, forceindex=False):
...
 
         Parameters
         ----------
         labels : str or sequence of strings
             Name of column(s) to be dropped.
-
+        axis : {0 or 'index', 1 or 'columns'}, default 0
+            Only axis=1 is currently supported.
+        errors : {'ignore', 'raise'}, default 'raise'
+            This parameter is currently ignored.
```
Comment (mrocklin) [inline python/cudf/dataframe/dataframe.py]: I recommend that we add an error if the user provides a non-default argument

```python
if errors != 'raise':
    raise NotImplementedError("The errors= keyword is not yet implemented")
```

**Comment by mrocklin** [comment]
It would be useful to add a tiny test for this, maybe here?

https://github.com/rapidsai/cudf/blob/cffacafd38757c6882d9d2c86993946572629592/python/cudf/tests/test_dataframe.py#L1593-L1601

You could probably add a line like the following to the parametrization

```python
lambda df: df.drop('x', axis=1, errors='raise')
``

**Comment by mrocklin** [comment]
The cudf process also asks that you include an entry in the changelog present at `cudf/CHANGELOG.md`

File: python/cudf/dataframe/dataframe.py [hunk 2]
```diff
@@ -722,7 +722,7 @@ def index(self):
     @index.setter
     def index(self, _index):
         if isinstance(_index, cudf.dataframe.multiindex.MultiIndex):
-            if len(_index) != len(self):
+            if len(_index) != len(self[self.columns[0]]):
```
Comment (ayushdg) [inline python/cudf/dataframe/dataframe.py]: @rjzamora Could you update your branch with the latest changes from cudf. New changes recently got merged.

**Comment by kkraus14** [comment]
add to whitelist

**Comment by kkraus14** [comment]
@rjzamora looks like a minor style issue:

```
python/cudf/dataframe/dataframe.py:961:80: E501 line too long (83 > 79 characters)
```

**Pull request merged**